client/web/src/components/app.tsx

```diff
@@ -1,35 +1,37 @@
+import cx from "classnames"
 import React from "react"
-import { Footer, Header, IP, State } from "src/components/legacy"
-import useAuth, { AuthResponse } from "src/hooks/auth"
+import LegacyClientView from "src/components/views/legacy-client-view"
```

Comment:
nit: since you added the "views" directory, the "-view" in the filename is kind of redundant now. Though I guess maybe there's typically consistency between the naming of the exported type `LegacyClientView` and the filename?

Reply:
Yeah hahaha I'm going really back and forth on these names. I like the "Client" part, but could also just be "LegacyView". I do think the file names should mirror the default component's name, so will leave as-is. And we can rename as we keep forming our understanding of the structures.